components/formats-gpl/src/loci/formats/in/LIFReader.java

```diff
@@ -1947,7 +2020,7 @@ else if (unit.equals("m")) {
             if (ms.sizeZ == 1) {
               ms.sizeZ = len;
               bytesPerAxis.put(nBytes, "Z");
-              physicalSizeZ = (physicalLen * len) / (len - 1);
+              physicalSizeZ = physicalLen;
```

Comment:
Thanks @michaelgoelzer. With the latest state of this PR, all [automated test failures](https://ci.openmicroscopy.org/job/BIOFORMATS-DEV-merge-repository-subset/377/) against our existing Leica LIF samples are related to physical size differences in X and Y which need to be adjusted on our side.

Re pixel size sample files, I found a couple of Leica LIF files (`curated/leica-lif/oliver/Multi-channel Measurements.lif`) in our repository with mismatching X/Y dimensions. We'll use them for testing this PR and let you know if we need additional files.